app/styles/utils.scss

```diff
@@ -14,3 +14,19 @@ h4,
 h5 {
   margin-top: 0;
 }
+
+@mixin clearfix() {
+  &:after { 
+     content: " ";
```

Comment:
Line should be indented 4 spaces, but was indented 5 spaces<br>Properties should be ordered clear, content, display, height